classes/emr/class.emrLongMenuAnswerOptionList.php

```diff
@@ -10,31 +10,21 @@
  *
  * @package    Plugins/ExcelMatrixResults
  */
-class emrLongMenuAnswerOptionList implements emrAnswerOptionList, Iterator
+class emrLongMenuAnswerOptionList extends emrAnswerOptionListAbstract implements emrAnswerOptionList, Iterator
 {
     use emrAnswerOptionListIterator;
     
     /**
      * @var assLongMenu
      */
-    protected $questionOBJ;
+    protected assQuestion $questionOBJ;
```

Comment:
Pls remove doc

Reply:
Would like to keep the doc in this and other extensions of emrAnswerOptionListAbstract since the base class requires assQuestions but the extension classes deal with specific question types.